src/v2/view-builder/views/captcha/CaptchaHelper.js

```diff
@@ -40,21 +25,28 @@ export function renderCaptcha(captchaConfig, form, onCaptchaSolvedCallback) {
   const captchaObject = captchaConfig.type === 'HCAPTCHA' ? hcaptcha : grecaptcha;
```

Comment:
pass the `captchaObject` through event as well ?

Reply:
I felt that it was a little redundant to pass in `captchaObject` as well since it could be easily derived from `captchaConfig`. Also, I wanted to keep the method as simple as possible with less args (easier for the consumer to use). However, if there are other concerns I can make this change.

Reply:
Mainly I don't like redundancy (`captchaConfig.type === 'HCAPTCHA' ? hcaptcha : grecaptcha` been run twice in separated places)
Maybe we could insert `captchaObject` into `captchaConfig` ?

Reply:
Ok I understand. Just a note with the latest changes, this is the only place where we perform this check now. The other one was removed during the refactoring today. 

I like the second idea better of inserting `captchaObject` into `captchaConfig` but I'm not sure if there's really any benefit since we would just offload this check from this file to another file.

Reply:
oh yeah, I think we're fine now.